packages/next-intl/package.json

```diff
@@ -57,6 +57,7 @@
     "dist",
     "server.d.ts",
     "client.d.ts",
+    "navigation.d.ts",
```

Comment:
I think this is why it was telling my I was missing types with the new next-intl/navigation import should be fixed with this.

Reply:
Very likely, yes! I also noticed this while incorporating localized pathnames in an app I'm working on. The latest RSC beta has this fixed.